charts/internal/calico/templates/multus/daemonset-multus-cni.yaml

```diff
@@ -0,0 +1,136 @@
+  selector:
+    matchLabels:
+      k8s-app: multus
+  updateStrategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 2
+  template:
+    metadata:
+      labels:
+        node.gardener.cloud/critical-component: "true"
+        networking.gardener.cloud/to-public-networks: allowed
+        networking.gardener.cloud/to-apiserver: allowed
+        networking.gardener.cloud/to-dns: allowed
+        k8s-app: multus
+        gardener.cloud/role: system-component
+    spec:
+      nodeSelector:
+        kubernetes.io/os: linux
+      hostNetwork: true
+      hostPID: true
+      tolerations:
+        # Make sure multus gets scheduled on all nodes.
+        - effect: NoSchedule
+          operator: Exists
+        - effect: NoExecute
+          operator: Exists
+      serviceAccountName: multus
+      securityContext:
+        seccompProfile:
+          type: RuntimeDefault
+      # Minimize downtime during a rolling upgrade or deletion; tell Kubernetes to do a "force
+      # deletion": https://kubernetes.io/docs/concepts/workloads/pods/pod/#termination-of-pods.
+      terminationGracePeriodSeconds: 0
+      priorityClassName: system-node-critical
+      initContainers:
+      # This container installs the multus CNI binaries
+      # and CNI network config file on each node.
+      - name: install-multus-cni
+        image: {{ index .Values.images "multus-cni" }}
+        command: ["/install_multus"]
+        args:
+          - "--type"
+          - "thin"
+        resources:
+          requests:
+            cpu: 10m
+            memory: 15Mi
+        volumeMounts:
+          - name: cnibin
+            mountPath: /host/opt/cni/bin
+            mountPropagation: Bidirectional
+        securityContext:
+          privileged: true
+      {{ if .Values.config.multus.installCNIPlugins }}
+      # This container installs the CNI binaries on each node.
+      - name: install-cni-plugins
+        image: {{ index .Values.images "cni-plugins" }}
+        command: ["/bin/sh"]
+        args:
+          - "-c"
+          - "cp /usr/src/cni/bin/* /host/opt/cni/bin/"
+        resources:
+          requests:
+            cpu: 10m
+            memory: 15Mi
+        volumeMounts:
+          - name: cnibin
+            mountPath: /host/opt/cni/bin
+        securityContext:
+          privileged: true
+      {{ end }}
+      containers:
+        # Runs multus container on each Kubernetes node.
+        - name: multus
+          image: {{ index .Values.images "multus-cni" }}
+          command: ["/thin_entrypoint"]
+          args:
+            - --cleanup-config-on-exit
+            - --cni-bin-dir=/host/opt/cni/bin
+            - --cni-conf-dir=/host/etc/cni/net.d
+            - --cni-version=0.3.1
+            - --multus-autoconfig-dir=/host/etc/cni/net.d
+            - --multus-cni-conf-dir=/etc/cni/multus/net.d
+            - --multus-conf-file=auto
+            - --multus-log-file=/var/log/multus.log
+            - --multus-log-level=error
```

Comment:
Is the log level sufficient? I am wondering whether we need more than just errors. However, we do not want the logs to be spammed. I am not sure which log levels are available here and what the recommended option would be. Could you please elaborate?